extensions/profiler/package.json

```
@@ -8,7 +8,7 @@
   "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
   "icon": "images/extension.png",
   "engines": {
-    "vscode": "0.10.0",
+    "vscode": ">=0.10.0",
```

I'd just make this * - it doesn't make much sense to have a vs code version for this extension anyway now that we have an azdata engine version.
```suggestion
    "vscode": "*",
```

---

Updated it to * to include all versions.